src/dropdown.test.interactions.filterable.ts

```diff
@@ -665,7 +625,7 @@ it('updates its input field when made filterable', async () => {
   expect(input?.value).to.equal('One');
 });
 
-it('clears its input field when `multiple` is set programmatically', async () => {
+it('clears its input field when multiselect is set programmatically', async () => {
```

Comment:
Tests were previously a mix of "\`multiple\`" and "multiselect". I change all of them to "multiselect". Also changed instances of "not \`multiple\`" to "single-select".